packages/calling-stateful-client/src/InternalCallContext.ts

```diff
@@ -25,24 +25,21 @@ export type RenderStatus = 'NotRendered' | 'Rendering' | 'Rendered' | 'Stopping'
 /**
  * Internal container to hold common state needed to keep track of renders.
  */
-export interface RenderInfo {
+export interface RenderInfo<T> {
```

Comment:
Why do we need to add this generic here? is this because they dont have typing for the two different kinds of streams and we know they can be different?

Reply:
No functional change here, just moved the stream from being defined inside the LocalRenderInfo/RemoteRenderInfo type to be being defined in the RenderInfo so if the RenderInfo type is used as a generic it has access to stream